lms/djangoapps/course_blocks/management/commands/tests/test_generate_course_blocks.py

```diff
@@ -43,6 +45,20 @@ def _assert_courses_in_block_cache(self, *course_keys):
         for course_key in course_keys:
             self.assertTrue(is_course_in_block_structure_cache(course_key, self.store))
 
+    def _assert_courses_not_in_block_storage(self, *course_keys):
+        """
+        Assert courses don't exist in course block storage.
+        """
+        for course_key in course_keys:
+            self.assertFalse(is_course_in_block_structure_storage(course_key, self.store))
+
+    def _assert_courses_in_block_storage(self, *course_keys):
+        """
+        Assert courses exist in course block storage.
+        """
+        for course_key in course_keys:
+            self.assertTrue(is_course_in_block_structure_storage(course_key, self.store))
```

Comment:
yeah - I've run into this a few times in tests.  One perspective on this is that, upon failure, it's better to have an error message say that it expected True but got a False value.  If we compare with a boolean, then the error message is not as rich.  Additionally, the way it is currently, the name of the helper method explicitly says the expectation rather than using an additional boolean - which is preferable from the CleanCode book standard.
But, I hear you.  Let me see if I can still DRY it any further.